packages/fxa-content-server/server/lib/routes/post-metrics.js

```diff
@@ -123,6 +123,8 @@ const BODY_SCHEMA = {
     })
     .optional(),
   numStoredAccounts: OFFSET_TYPE.min(0).optional(),
+  plan_id: STRING_TYPE.optional(),
+  product_id: STRING_TYPE.optional(),
```

Comment:
And can you add coverage for these to `tests/server/routes/post-metrics.js`?

Reply:
I don't see where these attributes are tested for in the `post-metrics.js` test, so adding them to an existing test is a bit of a head-scratcher for me.

Reply:
I was just thinking e.g. add them to these:

https://github.com/mozilla/fxa/blob/28cc9035ded8c6deaa9c9cea0affac3b60503623/packages/fxa-content-server/tests/server/routes/post-metrics.js#L92-L103

Just something to assert they make it through validation is all I had in mind.